prow/plugins/blockade/blockade.go

```diff
@@ -218,7 +218,7 @@ func compileApplicableBlockades(org, repo, branch string, log *logrus.Entry, blo
 			continue
 		}
 		// if branchregexp is specified, only consider blockades that apply to this branch
-		if raw.BranchRegexp != nil && !raw.BranchRe.MatchString(branch) {
+		if raw.BranchRe != nil && !raw.BranchRe.MatchString(branch) {
```

Comment:
Can we add a test that would have caught this?

Reply:
It's a bit unclear how we got into this state, I think maybe you have a regex that won't compile?

Reply:
but looking at https://github.com/kubernetes/test-infra/pull/21092/files, the regex there is valid ..?
and we appear to have tests that don't set this BranchRegexp field

Reply:
done